MarathonRecomp/patches/player_patches.cpp

```diff
@@ -116,7 +116,7 @@ bool PlayerDebugMode_RemapDebugExitButton(PPCRegister& r_PadState, PPCRegister&
 PPC_FUNC_IMPL(__imp__sub_82195500);
 PPC_FUNC(sub_82195500)
 {
-    if (!Config::PlayerDebugMode)
+    if (!(Config::PlayerDebugMode || Config::PlayerDemoCameraSwitch))
```

Comment:
Will wait for @hyperbx's opinion, but personally I think these two should be combined into a single option. I don't see much utility in having them available separately.